### DISCUSSION TITLE
Encoding for ZFTP profile

### DISCUSSION THREAD

**Discussion by jhonatan1090** [zowe/zowe-explorer-vscode#2045] [Q&A]
How can i change the codepage to upload a file to USS using UTF8?

I'am using ZFTP profile

Thanks!!

**Comment by FALLAI-Denis** [comment]
Hi @jhonatan1090 

Management of FTP transfer parameters is not provided in plugin FTP for Zowe CLI.
This also concerns the management of the encoding at the time of the transfer.

See
- [Use an "options file" when transferring a file](https://github.com/zowe/zowe-cli-ftp-plugin/issues/68)
-  [Add the management of codepages conversions](https://github.com/zowe/zowe-cli-ftp-plugin/issues/83)
